pkg/build/oci/oci.go

```diff
@@ -340,17 +350,17 @@ func publishIndexWithMediaType(mediaType ggcrtypes.MediaType, imgs map[types.Arc
 		img := imgs[arch]
 		mt, err := img.MediaType()
 		if err != nil {
-			return name.Digest{}, fmt.Errorf("failed to get mediatype: %w", err)
+			return name.Digest{}, idx, fmt.Errorf("failed to get mediatype: %w", err)
```

Comment:
Why not return `nil` here instead of `idx`? Are callers supposed to do anything with the returned `idx` value if there's an error?

(Studiously resisting a rant about error handling conventions in Go...)

Reply:
Good point I'll change it in a follow  up :+1: